src/App.js

```diff
@@ -1,29 +1,26 @@
-import logo from './logo.svg';
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import NavBar from './components/NavBar';
+import Book from './components/Book';
+import Categories from './components/Categories';
+import NoMatch from './components/NoMatch';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => (
+  <Router>
+    <NavBar />
+    <section className="content">
+      <Routes>
+        <Route path="/bookstore" element={<Book />} />
```

Comment:
- [ ] Kindly change the root path for your booklist to be `path="/"` this will ensure it serves as the homepage and you don't get the 404 page when you run the project.

Reply:
- [ ] @YussufOB You haven't changed the route as previously requested, kindly check on it.